src/artemis/external_interaction/tests/test_zocalo_handler.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import MagicMock, call
+
+from artemis.external_interaction.communicator_callbacks import FGSCallbackCollection
+from artemis.parameters import FullParameters
```

Comment:
Nit: Can we also test that the exceptions are thrown by the zocalo handler if ispyb has not been called correctly yet?